### DISCUSSION TITLE
What does the testing environment used by PR reviewers look like?

### DISCUSSION THREAD

**Discussion by latere-a-latere** [microsoft/winget-pkgs#1349] [General]
Following PR https://github.com/microsoft/winget-pkgs/pull/712, https://github.com/microsoft/winget-pkgs/pull/1067 and https://github.com/microsoft/winget-pkgs/pull/1195, I was wondering what the testing environment of the PR reviewers look like so I can hopefully set up a similar environment in a VM. Right now it appears that some manifests run fine on some machines while not running properly on others, so having a known spec could really help testing my branches before submitting them.

Does the standard Win10 Dev environment in HyperV suffice?

**Comment by chausner** [comment]
I can't answer your question but can recommend https://github.com/microsoft/winget-pkgs/pull/827 for testing.

**Comment by denelon** [comment]
@MarNicGit it's evolving pretty rapidly. We have been active with other teams inside of Microsoft to streamline the process. The environment you are suggesting might not have the same dependencies, but it is likely to catch many of the common errors we've been encountering. We have been working on getting #827 to a state where it can be used to help.
